src/Compilers/CSharp/Test/Emit2/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -5082,20 +5096,20 @@ public void Property_ChangeReturnType()
         {
             using var _ = new EditAndContinueTest()
                 .AddBaseline(
-                    source: $$"""
+                    source: MetadataUpdateDeletedAttributeSource + """
```

Comment:
Would appending `MetadataUpdateDeletedAttributeSource` at the end instead have improved the diff? Something to consider given that these tests are not easy to review...

Reply:
Possibly. Next time...